data/hai/hai news.txt

```diff
@@ -132,6 +226,45 @@ news "human citizen in Hai space"
 		word
 			`"`
 
+news "human reaction to Greenwater outfitter"
+	location
+		government "Hai"
+		planet "Greenwater"
+	to show
+		has "event: outfitter on Greenwater"
+	name
+		word
+			"Merchant captain"
+			"Engineer"
+			"Merchant pilot"
+	message
+		word
+			`"`
+		word
+			"I've been stranded here for ages waiting for the right parts to fix my ship. Finally I don't have to stay at that dank and musty hotel."
+			"I was weighting the need for Hai technology on my ship due to their high prices. Thankfully I've been able to outfit it with human tech so I don't have to pay such great amounts!"
+			"I had a good thing going before this outfitter showed up. I could transport human ship parts before there was any competition at all, now my services are practically useless!"
+		word
+			`"`
+
+news "businessperson in hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Businiessperson"
+			"Professional investor"
+			"Retiree"
+	message
+		word
+			`"`
+		word
+			"Real estate values for Greenwater homes have skyrocketed in recent months, so I'm taking advantage of this opportunity as much as I can!"
```

Comment:
This makes the assumption that the Hai allow US style price speculation on property and renter's markets. Something which is a legacy of rapid population growth and immigration. This isn't even true of all of our world, let along an alien world that seems to have resolved the majority of its economic difficulties and has been declining in population slowly for longer than humans have had fire.